clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -171,6 +176,7 @@ class device_info {
   size_t _local_mem_size;
   size_t _max_nd_range_size[3];
   int _max_nd_range_size_i[3];
+  size_t _max_stream_buffer_size;
```

Comment:
I think sycl::stream is sw concept,  not directly link to device.  so let's not introduce the helper function.  Just migrate use constant value eg (int_max).